src/components/ContentHome/styled.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components'
+import czechtech20191 from '../../assets/Home/czechtech2019-1.jpg'
+
+export const Wrapper = styled.article`
+	display: grid;
+	grid-template-columns: 1fr 6fr 1fr;
+	margin: 1em;
+	justify-items: center;
+
+	@media (max-width: 50em) {
+		margin: 0;
+		padding-top: 3em;
+		padding-bottom: 1em;
+	}
+`
+
+export const HomeSection = styled.section`
+	grid-column: 2;
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: center;
+	gap: 2em;
+	max-width: 60em;
+	margin: 1em;
+
+	@media (max-width: 50em) {
+		flex-flow: column wrap;
+		justify-items: center;
+		gap: 1em;
+	}
+`
+
+export const HomeSectionMultipleItems = styled(HomeSection)`
+	background-color: ${({ theme }) => theme.bg.secondary};
+	padding: 2em;
+
+	@media (max-width: 110em) {
+		flex-wrap: wrap;
+	}
+
+	@media (max-width: 30em) {
+		padding: 1em;
+		margin: 0 auto;
+	}
+`
+
+export const HomeSubsection = styled.div``
+
+export const HomeParagraph = styled.p`
+	font-size: 1rem;
+	line-height: 1.5em;
+`
+
+// TODO: If theme is light theme, HomeSubtitle should have different color
```

Comment:
Is this TODO still necessary? :thinking:

Reply:
Removed! :+1: